src/cmake_plugins.cmake

```diff
@@ -28,8 +28,12 @@ function(prepare_rssguard_plugin plugin_target_name)
     ${LIBRSSGUARD_SOURCE_PATH}
   )
 
-  if(WIN32 OR OS2)
+  if(MSVC OR OS2)
     install(TARGETS ${plugin_target_name} DESTINATION plugins)
+  elseif(MINGW)

```

Comment:
Respect the original intended folder for plugins on Windows OS. This change is not needed nor wanted.